test/integration/consumer_test.exs

```diff
@@ -109,9 +109,6 @@ defmodule ExRabbitPool.ConsumerTest do
       assert :ok = RabbitMQ.publish(channel, "#{queue}_exchange", "", "Hello Consumer!")
       assert_receive {:trace, ^pid, :receive, {:basic_deliver, "Hello Consumer!", _}}, 1000
 
-      assert_receive {:io_request, ^pid, _,
```

Comment:
If we are removing this assert, can you also remove the `Process.group_leader(pid, self())` from this test

Reply:
Done.